autoload/slime.vim

```diff
@@ -357,7 +357,7 @@ function! slime#send_cell(cell_delimiter) abort
   if !line_end
       let line_end = line("$")
   endif
-  call slime#send_range(line_ini, line_end)
+  call slime#send_range(line_ini+1, line_end-1)
```

Comment:
hi @mdedonno1337,

doing it unconditionally, it's off-by-one for the cases from line 354-359 — it would grab from line 2 and/or line (last - 1)

it would be more solid if the logic checked:
- if line_ini contains the cell_delimiter, +1
- if line_end contains the cell_delimiter, -1

alternatively, this could be done with else clauses for 354 and 357 🤔

Reply:
mmmm good point, didn't thought about not using a delimiter for the top/bottom...
it's my first vim script, will try to do it!

thanks for the feedback!